src/System.ComponentModel.TypeConverter/tests/SampleClasses.cs

```diff
@@ -82,7 +82,13 @@ public MyTypeListConverter(Type[] types)
         }
     }
 
+#if FUNCTIONAL_TESTS
     [TypeConverter("System.ComponentModel.Tests.BaseClassConverter, System.ComponentModel.TypeConverter.Tests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
+#elif PERFORMANCE_TESTS
```

Comment:
This is the only change to test code to allow compilation for System.ComponentModel.TypeConverter